client/app/components/edit-service-modal/edit-service-modal.html

```diff
@@ -7,22 +7,18 @@ <h4 class="modal-title" id="myModalLabel" translate>Edit Service</h4>
 <div class="modal-body">
   <form class="form-horizontal">
     <div pf-form-group pf-label="{{'Name'|translate}}" required>
-      <input id="name" name="name"
-             ng-model="vm.modalData.resource.name" type="text" required/>
+      <input id="name" name="name" ng-model="vm.modalData.name" type="text" required/>
     </div>
     <div pf-form-group pf-input-class="col-sm-9" pf-label="{{'Description'|translate}}">
-        <textarea id="description" name="description" ng-model="vm.modalData.resource.description">
-          {{ vm.modalData.resource.description }}
+        <textarea id="description" name="description" ng-model="vm.modalData.description">
+          {{ vm.modalData.description }}
         </textarea>
     </div>
+    <modal-actions
+      modal-data="vm.modalData"
+      on-cancel="vm.cancel()"
+      on-reset="vm.reset($event)"
```

Comment:
This is just a convention I have seen for naming component outputs. The naming is inline with how Angular2 does it. The intent is to make it simpler to migrate in the future.